### PR TITLE
Add support for projects targeting netcoreapp2.0

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   <PropertyGroup>
-    <VersionPrefix>1.0.3</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
 
     <!--
       The netstandard1.0 TFM doesn't actually compile. It's just there so Internal.AspNetCore.Sdk can take a dependency
       on this project.
     -->
-    <TargetFrameworks>netcoreapp1.1;net46;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net46;netstandard1.0</TargetFrameworks>
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj]: So why does this still have net46?

File: src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.nuspec [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <files>
     <file src="build\**" target="build\" />
     <file src="$publishDir$\net46\**" target="tools\net46\" />
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.nuspec]: This?

File: test/ApiCheck.Test/ApiCheck.Test.csproj [hunk 3]
```diff
@@ -3,8 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net46</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
```
Comment (Eilon) [inline test/ApiCheck.Test/ApiCheck.Test.csproj]: Still net46?

File: test/ApiCheck.Test/ApiCheck.Test.csproj [hunk 4]
```diff
@@ -3,8 +3,8 @@
...
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
```
Comment (Eilon) [inline test/ApiCheck.Test/ApiCheck.Test.csproj]: This needed?

File: src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj [hunk 5]
```diff
@@ -41,10 +41,6 @@
     <ProjectReference Include="..\NugetReferenceResolver\NugetReferenceResolver.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">
-    <PackageReference Include="System.Runtime.Loader" Version="$(CoreFxVersion)" />
```
Comment (dougbu) [inline src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj]: To confirm: This package is in .NET Core 2.0?

Comment (pranavkm) [inline src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj]: Yup - it's a feature of M.N.A.

**Pull request merged**